10/chapter-06.srt

```diff
@@ -224,6 +224,9 @@ So I will print current stanza here.
 00:04:19,330 --> 00:04:25,290
 And maybe the reps as well so we can check it.
 
+
+4:25 - Drew: Sure.
```

Comment:
I just merged this with the previous caption: https://github.com/fromPeerToPeer/captions/pull/75/commits/23eb81a1a0eb39bc2aac2af1ad36e931ea4b38f8